docusaurus/website/src/pages/styles.module.css

```diff
@@ -42,5 +42,8 @@
 
 .lightBackground {
   background-color: #f7f7f7;
-  color: rgb(31, 34, 39);
+}
+
+html[data-theme='dark'] .lightBackground {
```

Comment:
In a larger project I would prefer to put the colors in variables but for such a small file its ok. Just mentioning it because styles always starts out small and nimble then in many projects tend to get out of control over time.

Reply:
Is there another way we can replace either `.lightBackground` with another class, instead of just adding extra meta information for when its a dark theme?

For example, maybe change the class name to just `.gettingStartedSection` or more generic class name like `.featureSection` otherwise it may be difficult to track this change.

Reply:
I can imagine a scenario down the road where you may want to add an additional theme, or people are looking for examples to use in their own site, and this comes across as possibly bad practice. Idk :)